mihomo-party: fix

miwu04 opened the pull request:
Topic Description
-----------------

- mihomo-party: optimize redundant code
- Remove pnpm add mihomo-party/sysproxy-linux-{arch}-gnu as it is optional.
- Remove pnpm prepare as it's redundant with pnpm install.
- No need to manage the distribution of automatic construction equipment.
- mihomo-party: remove Mihomo upgrade interface and use system Mihomo
- Remove UI entry to upgrade Mihomo.
- Use system mihomo executable instead of the bundled copy.
- Drop support for mihomo-alpha as we do not ship it.
- mihomo-party: fix build script

Package(s) Affected
-------------------

- mihomo-party: 1.5.12-1

Security Update?
----------------

No

Build Order
-----------

```
#buildit mihomo-party
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`

miwu04 committed:
mihomo-party: fix build script

miwu04 committed:
mihomo-party: remove Mihomo upgrade interface and use system Mihomo
- Remove UI entry to upgrade Mihomo.
- Use system mihomo executable instead of the bundled copy.
- Drop support for mihomo-alpha as we do not ship it.

MingcongBai commented on app-network/mihomo-party/autobuild/prepare:
```diff
@@ -0,0 +1,2 @@
+abinfo "Prevent Corepack from showing the URL ..."
```
Uh... what?

MingcongBai commented on app-network/mihomo-party/autobuild/prepare:
```diff
@@ -0,0 +1,2 @@
+abinfo "Prevent Corepack from showing the URL ..."
+export COREPACK_ENABLE_DOWNLOAD_PROMPT=0
```
I'll add a handle for this in Autobuild. Drop this file.

miwu04 committed:
mihomo-party: optimize redundant code
- Remove pnpm add mihomo-party/sysproxy-linux-{arch}-gnu as it is optional.
- Remove pnpm prepare as it's redundant with pnpm install.
- No need to manage the distribution of automatic construction equipment.
